src/components/image-manager/advanced-pull.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react';
+
+import { Button, TextField, withStyles } from '@material-ui/core';
+
+const styles = theme => ({
+  root: {
+    display: 'flex'
+  },
+  imagesListField: {
+    margin: theme.spacing.unit,
+    width: '30ch'
+  },
+  pullImagesButton: {
+    margin: theme.spacing.unit
+  }
+});
+
+const defaultImagesList = [
+  'openchemistry/chemml:0.6.0',
+  'openchemistry/nwchem:6.6',
+  'openchemistry/psi4:1.2.1',
+  'openchemistry/torchani:1.2',
+];
```

Comment:
It seems like this list is also defined on the server? Should be just have an endpoint to return it? So we don't have to update it in two places?

Reply:
These are actually not on the server - not yet at least. We might add a new endpoint called "codes" where we have default codes listed on the server. But it hasn't been added yet.